### ISSUE TITLE
VSCode Insiders still not working

### ISSUE THREAD

**Issue by MSathieu** [desktop/desktop#3181]
After I updated to the newest version of Desktop, Insiders is still not recognised as an editor while it should work after #3012.

**Comment by lee-dohm** [comment]
Thanks for taking the time to contribute!

We noticed that this is a duplicate of https://github.com/desktop/desktop/issues/3011. Please follow that issue for further updates.
